modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,603 @@
+[source,yaml]
+----
+apiVersion: tekton.dev/v1
+kind: TaskRun
+metadata:
+  name: test-sidestep
+spec:
+  taskRef:
+    name: side-step
+  stepSpecs:
+    - name: test
+      computeResources:
+        requests:
+          memory: 1Gi
+  sidecarSpecs:
+    - name: side
+      computeResources:
+        requests:
+          cpu: 100m
+        limits:
+          cpu: 500m
+----
+
+[id="Operator-new-features-1-16_{context}"]
+=== Operator
+
+* With this update, {pipelines-shortname} includes the `git-clone` `StepAction` definition for a step that clones a Git repository. Use the HTTP resolver to reference this definition. The URL for the definition is `https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml`. The `StepAction` definition is also installed in the `openshift-pipelines` namespace, however, the cluster resolver does not support `StepAction` definitions at present.
++
+.Example usage of the `git-clone` step action in a task
+[source,yaml,subs="attributes+"]
+----
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: clone-repo-anon
+spec:
+  params:
+  - name: url
+    description: The URL of the Git repository to clone
+  workspaces:
+  - name: output
+    description: The git repo will be cloned onto the volume backing this Workspace.
+  steps:
+  - name: clone-repo-anon-step
+    ref:
+      resolver: http
+      params:
+      - name: url
+        value: https://raw.githubusercontent.com/openshift-pipelines/tektoncd-catalog/p/stepactions/stepaction-git-clone/0.4.1/stepaction-git-clone.yaml
+    params:
+    - name: URL
+      value: $(params.url)
+    - name: OUTPUT_PATH
+      value: $(workspaces.output.path)
+----
+
+* With this update, the `openshift-pipelines` namespace includes versioned tasks alongside standard tasks. For example, there is a `buildah` standard task and a `buildah-1-16-0` versioned task. While the standard task might be updated in subsequent releases, the versioned task remains exactly the same as it was in a specified version, except for correction of errors.
+
+* With this update, you can configure the `FailurePolicy`, `TimeoutSeconds`, and `SideEffects` options for webhooks for several components of {pipelines-shortname} by using the `TektonConfig` CR.
++
+.Example configuration of webhooks options for the `pipeline` component
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1alpha1
+kind: TektonConfig
+metadata:
+  name: config
+spec:
+  pipeline:
+    options:
+      webhookConfigurationOptions:
+        validation.webhook.pipeline.tekton.dev:
+          failurePolicy: Fail
+          timeoutSeconds: 20
+          sideEffects: None
+        webhook.pipeline.tekton.dev:
+          failurePolicy: Fail
+          timeoutSeconds: 20
+          sideEffects: None
+----
++
+You can use similar configuration for webhooks in the `triggers`, `pipelinesAsCode`, and `hub` components.
```

Comment:
Could this sentence be added to the first paragraph as it seems buried under the YAML. The format of other notes are in a paragraph block style.